CourseChamp/src/main/java/ca/mcgill/ecse428/CourseChamp/service/ReviewService.java

```diff
@@ -0,0 +1,37 @@
+package ca.mcgill.ecse428.CourseChamp.service;
+
+import ca.mcgill.ecse428.CourseChamp.dto.ReviewResponseDto;
+import ca.mcgill.ecse428.CourseChamp.exception.CourseChampException;
+import ca.mcgill.ecse428.CourseChamp.model.Review;
+import ca.mcgill.ecse428.CourseChamp.repository.ReviewRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+@Service
+public class ReviewService {
+
+    @Autowired
+    private ReviewRepository reviewRepository;
```

Comment:
PLEASE NOTE: ReviewRepository is an interface and cannot be instantiated...
It does not cause an error now because Java is assuming you will instantiate it using a class that implements the interface, but I am not sure which class that is.